packages/quantic/force-app/examples/main/lwc/exampleQuanticTimeframeFacet/exampleQuanticTimeframeFacet.js

```diff
@@ -0,0 +1,41 @@
+import {api, LightningElement, track}  from 'lwc';
+
```

Comment:
So far we added JSDoc comments only for the components of the main package. Since the example components are intended to be used solely in the examples community, I don't think having JSDoc comments on them would bring much value.